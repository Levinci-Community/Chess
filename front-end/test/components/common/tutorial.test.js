import React from 'react';
import renderer from 'react-test-renderer';
import Tutorial from '../../../src/components/common/tutorial';

describe('Components > Tutorial', () => {
  it('renders a snapshot for Tutorial using renderer', () => {
    const tree = renderer.create(
        <Tutorial />
    ).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
