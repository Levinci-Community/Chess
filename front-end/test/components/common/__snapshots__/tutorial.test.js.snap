// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components > Tutorial renders a snapshot for Tutorial using renderer 1`] = `
<div
  className="css-2rlxtj"
>
  <div
    className="react-joyride"
  />
</div>
`;
